vpr/src/base/read_options.cpp

```diff
@@ -1568,13 +1568,19 @@ argparse::ArgumentParser create_arg_parser(std::string prog_name, t_options& arg
 
     place_grp.add_argument(args.pad_loc_file, "--fix_pins")
         .help(
-            "Fixes I/O pad locations during placement. Valid options:\n"
+            "Fixes I/O pad locations randomly during placement. Valid options:\n"
             " * 'free' allows placement to optimize pad locations\n"
-            " * 'random' fixes pad locations to arbitraray locations\n"
-            " * path to a file specifying pad locations (.place format with only pads specified).")
+            " * 'random' fixes pad locations to arbitrary locations\n.")
         .default_value("free")
         .show_in(argparse::ShowIn::HELP_ONLY);
 
+    place_grp.add_argument(args.constraints_file, "--fix_clusters")
+        .help(
+            "Fixes block locations during placement. Valid options:\n"
+            " * path to a file specifying block locations (.place format with block locations specified).")
+        .default_value("not_locked")
```

Comment:
Update this to just have a filename, default is empty string (no constraint file).